docs/content/how-to/querying-tables.md

```diff
@@ -51,6 +51,11 @@ SELECT * FROM t /*+ OPTIONS('scan.snapshot-id' = '1') */;
 -- read the snapshot from specified timestamp in unix milliseconds
 SELECT * FROM t /*+ OPTIONS('scan.timestamp-millis' = '1678883047356') */;
 
+-- read the snapshot from specified datetime
+-- Supported formats include：yyyy,yyyy-MM,yyyy-MM-dd, yyyy-MM-dd HH, yyyy-MM-dd HH:mm, yyyy-MM-dd HH:mm:ss, yyyy-MM-dd HH:mm:ss.SSS
+-- default local time zone,If you need to specify a time zone, please end with '/UTC{offset}',for example: 2023-12-11 12:12/UTC+8
+SELECT * FROM t /*+ OPTIONS('scan.datetime' = '2023-12-09 23:09') */;
```

Comment:
just `scan.timestamp`?

Reply:
Has been modified to `scan.timestamp` ,please recheck!

![image](https://github.com/apache/incubator-paimon/assets/40030020/21cd47bb-8d25-4bee-86fc-76b09f6d49c8)